### DISCUSSION TITLE
jsonSchema被动模式下$deps有值但$deps[0]为空

### DISCUSSION THREAD

**Discussion by a1945472906** [alibaba/formily#1726] [Q&A 2.x] [answered]
[https://codesandbox.io/s/jjgwm?file=/src/components/test.vue](url)

下拉选择checked_true后 可以看到$deps是有值的 但这里如果改成$deps[0]就为空了
预期下拉选择checked_true后 checkBox应该被勾选

**Comment by MisicDemone** [comment, marked as answer]
不响应的原因是你没有watch prop的变更。
![image](https://user-images.githubusercontent.com/10394516/124713401-18f57e00-df33-11eb-98c0-ddc218a9975b.png)

**Reply by a1945472906** [reply]
的确是这样 非常感谢！
